daemon.go

```diff
@@ -122,6 +121,7 @@ var (
 	flag_verbose         = flag.Bool("verbose", false, "Be verbose about which directories are watched.")
 
 	// initialized in main() due to custom type.
+	flag_directories   globList
 	flag_excludedDirs  globList
```

Comment:
This is the only line I have a gripe with.

`directories` is not a glob list and it never should be. I think we should refactor `globList` into a more general `stringList` and let `globList` build upon it.

Do you want to make that change? Let me know so I can do it instead.